set.mm

```diff
@@ -645565,9 +645707,21 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
       VMATUJVAVCVMUKULUMVGVOVJVIIVFBCPUNUOUPUQURUS $.
   $}
 
+  ${
+    dvdsexpad.1 $e |- ( ph -> A e. ZZ ) $.
+    dvdsexpad.2 $e |- ( ph -> B e. ZZ ) $.
+    dvdsexpad.3 $e |- ( ph -> N e. NN0 ) $.
+    dvdsexpad.5 $e |- ( ph -> A || B ) $.
+    $( Deduction associated with ~ dvdsexpim .  (Contributed by SN,
+       21-Aug-2024.) $)
+    dvdsexpad $p |- ( ph -> ( A ^ N ) || ( B ^ N ) ) $=
+      ( cdvds wbr cexp co cz wcel cn0 wi dvdsexpim syl3anc mpd ) ABCIJZBDKLCDKL
+      IJZHABMNCMNDONTUAPEFGBCDQRS $.
+  $}
+
   $( If ` A ` and ` B ` are relatively prime, then so are ` A ^ N ` and
-     ` B ^ N ` . ~ rppwr extended to nonnegative integers.  (Contributed by
-     Steven Nguyen, 4-Apr-2023.) $)
+     ` B ^ N ` . ~ rppwr extended to nonnegative integers.  Less general than
+     ~ rpexp12i .  (Contributed by Steven Nguyen, 4-Apr-2023.) $)
   nn0rppwr $p |- ( ( A e. NN0 /\ B e. NN0 /\ N e. NN0 ) ->
                     ( ( A gcd B ) = 1 -> ( ( A ^ N ) gcd ( B ^ N ) ) = 1 ) ) $=
```

Comment:
True for any $A, B \in \mathbb{Z}$.

Reply:
the comment should now mention https://us.metamath.org/mpeuni/rpexp12i.html

Reply:
Oh, sorry. So why add this theorem ? Isn't rpexp12i enough ?

Reply:
I didn't notice rpexp12i when I made nn0rppwr. Also nn0rppwr doesn't depend on ~ df-prime I guess, so it and the theorems that use it could be placed earlier